src/components/Settings/Speech/Speech.container.js

```diff
@@ -46,6 +49,17 @@ export class SpeechContainer extends Component {
 
   async componentDidMount() {}
 
+  handleUpdateElevenLabsApiKey = async apiKey => {
+    const { changeElevenLabsApiKey } = this.props;
+
+    if (apiKey && !validateApiKeyFormat(apiKey)) {
+      throw new Error('Invalid API key format');
+    }
```

Comment:
Need to be sure that the function that calls the `handleUpdateElevenLabsApiKey` is wrapped within a try catch. If not could break the app